django_telegrambot/apps.py

```diff
@@ -170,11 +169,12 @@ def ready(self):
                 logger.error('WEBHOOK_CERTIFICATE not found in {} '.format(cert))
 
         for b in bots_list:
-            token = b.get('TOKEN', None)
-            context = b.get('CONTEXT', False)
-            if not token:
-                break
+            bot_data = {
```

Comment:
I like the idea to have the token/name/id in a dict, but it also breaks with the previous style, while some other properties are still in own varaibles (like context, timeout and so on..). I think about adding them to the dict as well..

Reply:
on the other hand, they are used a lot in this scope, which makes it more readable and clear to have a simple variable for such properties.

Reply:
You also moved the bot object itself to the bot_data. So some of the most important data about of the bot is nested in dictionaries.. Do you think it's possible to flatten this structure to put all the additional data like tokens, names, ids as properties of the bot object, and keep the `cls.bots` list? Or is this a problem because the data needs to be present before the bots are initialized?

Reply:
It was actually what I wanted to do at first, but the problem is that the telegram Bot object has no property to store custom data... unless we create our own wrapper class for bots and add extra info property

Reply:
ok, I will open an Issue for that, to think about how to handle the bot settings in code. I thought about using bot specific settings anyways (to have tokens and config in the app not the project settings). But for now I merge it as it is.

Reply:
Actually I pushed new commits with the same thing I said (wrapper for the bot class). not a perfect solution but.. meh.